distribution/docs/src/main/resources/content/_running/manual-removal-expired.adoc

```diff
@@ -0,0 +1,38 @@
+:title: Manual Removal of Expired Records
+:type: dataManagement
+:status: published
+:summary: Methods of removing Expired Records from the Catalog.
```

Comment:
✏️ I don't think `Expired Records` needs caps/is a proper noun

Reply:
fixed